x-pack/plugins/cases/public/components/severity/config.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useEuiTheme, EuiHealth } from '@elastic/eui';
+import { CaseSeverity } from '../../../common/types/domain';
+import { CRITICAL, HIGH, LOW, MEDIUM } from './translations';
+
+interface Props {
+  severity: 'low' | 'medium' | 'high' | 'critical';
+  label: string;
+}
+
+export const severities = {
+  [CaseSeverity.LOW]: {
+    label: LOW,
+  },
+  [CaseSeverity.MEDIUM]: {
+    label: MEDIUM,
+  },
+  [CaseSeverity.HIGH]: {
+    label: HIGH,
+  },
+  [CaseSeverity.CRITICAL]: {
+    label: CRITICAL,
+  },
+};
+
+// eslint-disable-next-line react/display-name
+export const SeverityHealth: React.FC<Props> = ({ severity, label }) => {
```

Comment:
I think the label is also something that is known best here internally. This component should know what color-label combination should be shown depending on the severity

Reply:
Done, the label is not internally.